lib/jormungandr/test/integration/Cardano/Wallet/Jormungandr/NetworkSpec.hs

```diff
@@ -258,6 +282,27 @@ spec = do
             let outs = replicate 3 (head $ outputs txNonEmpty)
             let tx = (Tx (fragmentId inps outs []) inps outs, [])
             runExceptT (postTx nw tx) `shouldThrow` anyException
+
+        it "decodeExternalTx works ok with properly constructed (Tx, [TxWitness]) binary blob" $
+            \(_, nw) -> do
+                property $ \(SignedTx signedTx) -> monadicIO $ liftIO $ do
+                    let encode ((Tx _ inps outs), wits) = runPut
+                            $ withHeader MsgTypeTransaction
+                            $ putSignedTx inps outs wits
+                    let encodedSignedTx = BL.toStrict $ encode signedTx
+                    runExceptT (decodeExternalTx nw encodedSignedTx)
+                        `shouldReturn` Right signedTx
+
+        it "decodeExternalTx throws an exception when binary blob has non-transaction-type header \
+           \or is wrongly constructed binary blob" $
+            \(_, nw) -> do
+                property $ \(SignedTx signedTx) -> monadicIO $ liftIO $ do
+                    let encodeWrongly ((Tx _ inps outs), wits) = runPut
+                            $ withHeader MsgTypeInitial
+                            $ putSignedTx inps outs wits
+                    let encodedSignedTx = BL.toStrict $ encodeWrongly signedTx
+                    result <- runExceptT (decodeExternalTx nw encodedSignedTx)
+                    result `shouldBe` (Left $ ErrDecodeExternalTxWrongPayload "wrongly constructed binary blob")
```

Comment:
Suggest:
```hs
                    result `shouldBe`
                        (Left $ ErrDecodeExternalTxWrongPayload
                            "wrongly constructed binary blob")
```
https://github.com/input-output-hk/cardano-wallet/wiki/Coding-Standards#limit-line-length-to-80-characters

Reply:
done